sync-todo/v2/client/flutter/README.md

```diff
@@ -1,15 +1,24 @@
 # Realm Flutter Todo
 
-A todo list application built with the [Realm Flutter SDK](https://www.mongodb.com/docs/realm/sdk/flutter/)
-and [Atlas Device Sync](https://www.mongodb.com/docs/atlas/app-services/sync/).
+A todo list application built with the [Realm Flutter SDK](https://www.mongodb.com/docs/realm/sdk/flutter/) and [Atlas Device Sync](https://www.mongodb.com/docs/atlas/app-services/sync/).
+
+You can follow along with the [Flutter Tutorial](https://www.mongodb.com/docs/atlas/app-services/tutorial/flutter/) to see how to build, modify, and 
+run this template app.
 
 ## Configuration
 
-Ensure `flutter_todo/assets/config/atlasConfig.json` exists and contains the following properties:
+For this template app to work, you must ensure that `flutter_todo/assets/config/atlasConfig.json` exists and contains the following properties:
```

Comment:
This is a nitpick for some of these, but presumably the people who are consuming this README will be doing so either from the artifact repo (https://github.com/mongodb/template-app-dart-flutter-todo) or as a download from the Atlas UI where `flutter_todo` is not part of the directory structure. So I think when we're providing the path, we should do it relative to where the README is located and not make any assumptions about the enclosing directory structure.

```suggestion
For this template app to work, you must ensure that `assets/config/atlasConfig.json` exists and contains the following properties:
```

(Also, if you take this suggestion, we should change it in ln 39 down below, too, I just can't leave a comment there since we're not modifying it here.)